docs/variants/qemu_q35/development.md

```diff
@@ -6,10 +6,10 @@
 docker instance (host machine) using built OVMF firmware image.
 
 + QEMU Installation:
-    - [Install QEMU on your linux distro](<https://www.qemu.org/download/#linux>)
+    - [Install QEMU on your linux distro](https://www.qemu.org/download/#linux)
 
  ```bash
- qemu-system-x86_64 -drive if=pflash,format=raw,file=Build/OvmfX64/DEBUG_GCC5/FV/OVMF.fd -debugcon file:debug.log -global isa-debugcon.iobase=0x402 -global ICH9-LPC.disable_s3=1 -net none -machine q35,smm=on
+ qemu-system-x86_64 -drive if=pflash,format=raw,file=Build/OvmfX64/RELEASE_GCC5/FV/OVMF.fd -debugcon file:debug.log -global ICH9-LPC.disable_s3=1 -net none -machine q35,smm=on
```

Comment:
For development it is desirable to keep using DEBUG build. Why have you changed it to RELEASE? I haven't asked for that...

Reply:
I reverted the changes and fixed them accordingly.